local/dfm/lib/common.sh

```
@@ -0,0 +1,236 @@
+# @description Handle an error
+# @param $1 Line number
+# @param $2 Command
+# @return void
+lib::error::handle()
+{
+  local exit_code=$?
+  local line_no=$1
+  local command=$2
+
+  case $exit_code in
+    ${ERROR_CODES[INVALID_ARGUMENT]})
```

Shellcheck (0.10.0)

[warning] Quote expansions in case patterns to match literally rather than as a glob.

(SC2254)